### ISSUE TITLE
Add "people" section

### ISSUE THREAD

**Issue by sunahsuh** [mozilla/firefox-data-docs#138]
For lack of a more obvious place, docs.telemetry.mozilla.org seems like it'd be a decent place to put functional pods and the folks attached to them

**Comment by harterrt** [comment]
@kparlante @rjweiss - Does this sound good to you? Does this documentation exist somewhere now?

**Comment by georgf** [comment]
Erin Lancaster is working on a Mana page for this.
It is probably sufficient to wait on that, then make sure we have discoverable linking to it in the documentation.

**Comment by wlach** [comment]
We don't have functional pods anymore (though I have heard rumors of us bringing them back). Let's just close this, can always open a new issue later.
